libs/gltfio/src/extended/AssetLoaderExtended.h

```diff
@@ -70,7 +70,9 @@ struct AssetLoaderExtended {
         : mEngine(engine),
           mGltfPath(config.gltfPath),
           mMaterials(materials),
-          mUriDataCache(std::make_shared<UriDataCache>()) {}
+          mUriDataCache(std::make_shared<UriDataCache>()) {
+        mCgltfBuffersLoaded = false;
```

Comment:
please move this up to the constructor member initializer list.

Reply:
Done